src/site/stages/build/process-cms-exports/schemas/common/blockContent.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable import/no-dynamic-require */
+const { getAllSchemasOfType } = require('./helpers');
+
+const paragraphSchemas = getAllSchemasOfType('block_content');
+module.exports = {
+  $id: 'BlockContent',
+  anyOf: paragraphSchemas,
```

Comment:
Is this some kind of future-proofing?  At the moment it looks like we just have `block_content-alert.js`

Reply:
Yeah. This way we don't have to remember add each `block_content` schema in manually.

Though I see now I need to rename that variable. :sweat_smile: